scri/waveform_base.py

```diff
@@ -961,7 +961,7 @@ def interpolate(self, tprime):
         W.t = np.copy(tprime)
         W.frame = quaternion.squad(self.frame, self.t, W.t)
         W.data = np.empty((W.n_times,) + self.data.shape[1:], dtype=self.data.dtype)
-        W.data_2d[:] = CubicSpline(self.t, self.data_2d.view(float))(W.t).view(complex)
+        W.data_2d[:] = CubicSpline(self.t, self.data_2d.view(complex))(W.t).view(complex)
```

Comment:
I *think* I wrote this at a time when CubicSpline couldn't handle complex inputs.  But if it can nowadays, I suppose you can just remove both of these `.view`s, right?